packages/app/src/DBSearchPage.tsx

```diff
@@ -926,6 +1009,24 @@ function DBSearchPage() {
                   Alerts
                 </Button>
               )}
+              {!!savedSearch && (
+                <Tags
+                  allowCreate
+                  values={savedSearch.tags || []}
+                  onChange={handleUpdateTags}
+                >
+                  <Button
+                    variant="outline"
+                    color="dark.2"
+                    px="xs"
+                    size="xs"
+                    style={{ flexShrink: 0 }}
+                  >
+                    <i className="bi bi-tags-fill me-1"></i>
+                    {savedSearch.tags?.length || 0}
+                  </Button>
+                </Tags>
```

Comment:
Just a note, clicking on this "x" in the input does nothing 
![image](https://github.com/user-attachments/assets/d5dedbc3-3241-4a5b-9a12-0be9cf80f716)